src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -884,88 +898,141 @@ namespace HeatBalFiniteDiffManager {
                                 OutputProcessor::SOVStoreType::Summed,
                                 state.dataSurface->Surface(SurfNum).Name);
 
-            // Setup EMS Material Actuators
+            // Setup EMS Material Actuators for Conductivity and Specific Heat
             ConstrNum = state.dataSurface->Surface(SurfNum).Construction;
-            for (int mat = 1; mat <= state.dataConstruction->Construct(ConstrNum).TotLayers; ++mat) {
-                EnergyPlus::SetupEMSActuator(state,
-                                             "CondFD Surface Material Layer",
-                                             SurfaceFD(SurfNum).condMaterialActuators(mat).actuatorName,
-                                             "Thermal Conductivity",
-                                             "[W/m-K]",
-                                             SurfaceFD(SurfNum).condMaterialActuators(mat).isActuated,
-                                             SurfaceFD(SurfNum).condMaterialActuators(mat).actuatedValue);
-                EnergyPlus::SetupEMSActuator(state,
-                                             "CondFD Surface Material Layer",
-                                             SurfaceFD(SurfNum).specHeatMaterialActuators(mat).actuatorName,
-                                             "Specific Heat",
-                                             "[J/kg-C]",
-                                             SurfaceFD(SurfNum).specHeatMaterialActuators(mat).isActuated,
-                                             SurfaceFD(SurfNum).specHeatMaterialActuators(mat).actuatedValue);
+
```

Comment:
@mjwitte I rearranged this to break out the internal source and EMS source values on separate output variables. The EMS value is now the only one on the meter.

I'm also only protecting the setup of the actuators and output variables here with `AnyEnergyManagementSystemInModel` - it would be too big of a refactor to protect the rest of it with that flag.